File: payment-service/src/main/java/com/sudo248/paymentservice/controller/dto/PaymentDto.java
package com.sudo248.paymentservice.controller.dto;

import com.sudo248.paymentservice.repository.entity.PaymentStatus;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class PaymentDto {
    private String paymentId;
    private String orderId;
    private String orderType;
    private String bankCode;
    private Double amount;
    private String paymentType;
    private String ipAddress;
    private String paymentUrl;
    private PaymentStatus paymentStatus;
}

File: client/Soc/app/src/main/java/com/sudo248/soc/data/dto/payment/PaymentDto.kt
package com.sudo248.soc.data.dto.payment

data class PaymentDto(
    val paymentId: String = "",
    val paymentUrl: String? = null,
    val paymentType: String,
    val bankCode: String? = null,
    val orderId: String,
    val orderType: String = "100000",
)
